### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by Richez003**
Added a contact form to my portfolio page
Created a different branch
used the correct git workflows
checked for linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@ <h2 class="sd">
                 don’t hestiate to contact me.
             </p>
             <ul class="icon-list">
-                <li><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="https://twitter.com/richardfriday14" target="blank"><img src="./images/twitter.png" alt="twitter"></a></li>
```
Comment (Meltrust) [inline index.html]: - [x] On your form, all your input fields elements must be wrapped on the form element.  The submit button too.
- [x] Please, ensure the form works properly and sends the data correctly.

## Explanation

This is the reason why the form does not work properly.  You need to wrap all the elements of the form, inside the `<form>` tag.

## Hints

I managed to make it work like this:

```HTML

      <form
        class="form-style"
        action="https://formspree.io/f/mnqwzknj"
        method="POST"
      >
        <input type="text" maxlength="30" placeholder="Full Name" required />

        <input type="email" name="email" placeholder="Enter address" required />

        <textarea
          class="text"
          name="msg"
          maxlength="500"
          placeholder="Enter text here"
          required
        ></textarea>

        <div class="button-cont">
          <button type="submit" class="potfolio-but">Get in Touch</button>
        </div>
      </form>

```

Comment (Meltrust) [inline index.html]: According to the project requirements:

image

Therefore:

- [x] Please, kindly use Flexbox to align the elements of your form, as stated in the requirements.

That way, we will meet this particular requirement 💪

Comment (Meltrust) [reply index.html]: In the text "
        I'm always interested in hearing about new projects, so if you'd like to
        chat please get in touch.
      "

- [x] You have not implemented font styling for that paragraph.  Please add the correct font styling for this `<p>`.  

image





## Hints

Look at the Figma:

image

Please, implement these stylings to that `<p>`, so your project looks like the Figma 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Richez003,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice forms and styling✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Richez003) [reply index.html]: done

Comment (Richez003) [reply index.html]: done

File: index.html [hunk 2]
```diff
@@ -259,17 +259,17 @@ <h1>About Me</h1>
             
                 
                     <textarea class="text" name="msg" maxlength="500" placeholder="Enter text here" required></textarea>
-                
+                </form>
                 <div class="button-cont">
-                    <button type="submit" class="potfolio-but">Get it Touch</button>
+                    <button type="submit" class="potfolio-but">Get in Touch</button>
```
Comment (TedLivist) [inline index.html]: - [x] This submit button should be within the form element. Kindly move this submit button into the form, just before the `</form>` closing tag

File: index.html [hunk 3]
```diff
@@ -250,7 +250,7 @@ <h1>About Me</h1>
                 
                     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
```
Comment (TedLivist) [inline index.html:251]: - [x] You are yet to implement the font styling as requested previously. Kindly do that. Your font is quite different from what we have on the Figma template. Kindly take a look at the highlighted styles for hints on how to style your paragraph.

  Your Implementation | Figma Design
  :---:|:---:
  rich-font-his | rich-font

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Richez003,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Richez003) [reply index.html]: done

**Comment by Meltrust** [comment]
@Richez003 if you need help or you are blocked, please don't hesitate to contact me on Slack if you want.

File: index.html (hunk 3 above)
Comment (Richez003) [reply index.html:251]: done

Comment (Richez003) [reply index.html:251]: using "inter" font won't give that style, so I have changed it to "sans-serif;" and it gives the exact  same style.

**Review by phelian23 — APPROVED** [review]
Hi @Richez003, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**